test/path-utils.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { PathUtils } from "../src/classes/path-utils";
+
+it('gets default paths', () => {
+  expect((PathUtils.savePath())).toBeDefined();
```

Comment:
This just asserts that path is defined, could it be asserted that it actually equals something?